checkpoint/css/main.css

```
@@ -198,3 +209,35 @@ header img {
   object-fit: cover;
 }
 /* end header styles */
+
+/**
+ * Here we use multiple CSS selectors together. These are
+ * ID selector (hobbies)
+ * Descendent selector (the space between #hobbies and ul)
+ * Element selector (the ul and the li)
+ * Child selector (the > between ul and li)
+ * Pseudo-class selector (the :nth-child(odd))
+ *
+ * Explained in plain English this means:
+ * Select the odd <li> tags which are direct children
+ * of a <ul> tag, which is a descendant of the HTML element,
+ * which is marked with the ID #hobbies
+ */
+#hobbies ul > li:nth-child(odd) {
+  color: orangered;
+}
+
+#hobbies ul > li:nth-child(even) {
+  color: midnightblue;
+}
+
+/**
+ * Using e CSS Pseudo-Element Selector
+ * Check: https://redi-school.github.io/nrw-html-and-css-2021-spring/lesson20/#4-css-pseudo-elements
```

Looks like you missed the closing part of the comment
```suggestion
 * Check: https://redi-school.github.io/nrw-html-and-css-2021-spring/lesson20/#4-css-pseudo-elements
 */
```

---

Oh yes, thank you